app/javascript/components/patient/icons/BadgeHoH.js

```diff
@@ -9,7 +9,9 @@ class BadgeHoH extends React.Component {
       <React.Fragment>
         <span data-for={`${this.props.patientId}-hoh`} data-tip="" className={this.props.customClass}>
           <Badge variant="dark" className="badge-hoh">
-            <span>HoH</span>
+            {/* Maintaining a separate element prevents the screen reader from reading this as "Ho" */}
+            <span className="sr-only">Head of Household</span>
```

Comment:
Should this start with a space? All the other `sr-only` spans here start with a space, but also I don't understand why.

Reply:
I actually went ahead and removed this and replaced it with an aria-label. Not sure why i made this separately when a aria-label does the tricck